### DISCUSSION TITLE
Renovatebot dealing with PRs open more than X period of time

### DISCUSSION THREAD

**Discussion by alaamub** [renovatebot/renovate#11804] [Request Help]
Hi, does Renovatebot support auto-closing PRs that are not necessarily stale, but been open for X amount of time. 

For example, can I tell renovatebot to autoclose all PRs `age > 30  days` and delete their branches?

**Comment by rarkins** [comment]
No, that is not supported and I don't recall any feature request existing for it yet. It sounds like it could encourage bad habits ("ignore it long enough and it goes away") so I'd be interested to hear your thoughts on the benefits of this approach. E.g. is it for your personal projects or for team projects?

**Reply by alaamub** [reply]
totally, I agree with it "encourage bad habits". This is for team project, the concern was from infrastructure team, they want a backstop mechanism so we don't have year-old dangling branches that we aren't actioning.

**Reply by rarkins** [reply]
So far this doesn't sound like a feature I'd like to implement in Renovate. Perhaps there exists a "stale bot" you can use instead.

**Reply by alaamub** [reply]
Make sense, thanks @rarkins I appreciate the feedback!
